requirements/dev.txt

```diff
@@ -33,6 +33,14 @@ transifex-client==0.12.4
 # Dependency of transifex-client
 urllib3==1.20
 
+# Needed for documentation_crawler
+pathlib2==2.2.1
+#Needed for pathlib2==2.2.1
+scandir==1.5
+
```

Comment:
from looking at `pip` output, I think scandir/pathlib2 are in fact dependencies of ipython in Python 2, just not in Pyhton 3.  So we'll need them in prod in some circumstances.